test/cql-pytest/test_materialized_view.py

```diff
@@ -288,6 +288,18 @@ def test_is_not_null_forbidden_in_filter(cql, test_keyspace, cassandra_bug):
         finally:
             cql.execute(f"DROP MATERIALIZED VIEW IF EXISTS {test_keyspace}.{mv}")
 
+# Test that it is not allowed to create a materialized view with gc_grace_seconds=0.
+# This check is missing from Cassandra, so the test is marked as Scylla-only.
```

Comment:
Thanks. Please explain in the comment before the test - not just in the cover letter - why you consider it to be a bug in Cassandra, and use the fixture "cassandra_bug" - not "scylla_only". (and please also mention that this test reproduces the issue number)

If you feel like it, it will also be nice to open an issue on Cassandra explaining how what they implemented in CASSANDRA-9917 is inconsistent (checks only ALTER but not CREATE). Then you'll get the official stamp of approval that it's a Cassandra bug ;-) But it's not required, only do it if you want to be a good Cassandra citizen.